styles.css

```diff
@@ -0,0 +1,48 @@
+body {
+    background-color: #587aec;
+    color: #ffffff;
+}
+h1, h2, h3, p {
+    text-align: center;
+}
+.center-content {
+    text-align: center;
+}
+.center-table {
```

Comment:
this is a really bad way of centering a table 😅 . Lets keep centering content for the next 2 lessons.